tests/strings_multiline_negative/strings_multiline_negative.fz

```diff
@@ -0,0 +1,45 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of Fuzion test strings_multiline_negative
+#
+# -----------------------------------------------------------------------
+
+strings_multiline_negative is
+
+  x := 1
+
+  a := """"""    //  1. should flag an error: Expected multiline string to start in first line following fat quotation
+  b := """ """   //  2. should flag an error: Expected multiline string to start in first line following fat quotation
+
+                 //  3. should NYI flag NYI an NYI error: (error in line 32) Expected multiline string to start in first line following fat quotation
```

Comment:
This negative test case does not work since I can not put the comment on the line where the error occurs.